layouts/partials/footer.html

```diff
@@ -5,7 +5,7 @@
       {{ with site.GetPage "page" "docs/tutorials/stateless-application/hello-minikube" }}<a href="{{ .RelPermalink }}">{{ .LinkTitle }}</a>{{ end }}
       {{ $sections := slice "docs/home" "blog" "training" "partners" "community" "case-studies" }}
       {{ range $sections }}
-      {{ with site.GetPage "section" . }}<a class="text-white" href="{{ .RelPermalink }}">{{ .LinkTitle }}</a>{{ end }}
+      {{ with site.GetPage "section" . }}<a class="text-white" href="{{ .RelPermalink }}">{{if eq .LinkTitle "Home" }}Documentation{{else}}{{ .LinkTitle }}{{end}}</a>{{ end }}
```

Comment:
I'm afraid this change won't be right for localizations other than English.

We need a different fix. Try:
```suggestion
      {{ with site.GetPage "section" . }}<a class="text-white" href="{{ .RelPermalink }}">{{ .Title }}</a>{{ end }}
```

Reply:
On second thought, Splitting the text at whitespaces can still create problems for some languages, spanish for instance. Should we go with Kubernetes Documentation as a whole?

Reply:
“Kubernetes Documentation”  works for me.

We could also add a front matter field to the index page for `/docs/home`, eg `footerTitle`, and use that custom field if it is set.

Reply:
This might be some work for the the localization teams, If this is an issue with other languages as well. I just modified the front matter field 'linkTitle' to say "Documentation".

Reply:
(Your .Title approach can minimize localization workload)